entry/src/main/ets/MainAbility/pages/index.ets

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	MaterialSelect,
+	MenuOption,
+	MultipleSelect,
+	MultipleMenuOption,
+	DotMenu,
+	DotMenuOption
+} from '@ohos/material-select'
+
+
+@Entry
+@Component
+struct Index {
+	name: string = 'Industry'
+	placeholder: string = 'select'
+	selectMenu: MenuOption[] = []
+	selection: MenuOption = new MenuOption(null, null)
+	selectModel: MaterialSelect.Model = new MaterialSelect.Model($r('app.media.dropdown'), 'Industry', 'sel')
+	selectMenuModel: MaterialSelect.MenuModel = new MaterialSelect.MenuModel()
+	multipleSelectName: string = 'Use Case'
+	multipleSelectMenu: MultipleMenuOption[] = []
+	multipleSelectModel: MultipleSelect.Model = new MultipleSelect.Model($r('app.media.dropdown'), 'Use Case')
+	multipleMenuModel: MultipleSelect.MenuModel = new MultipleSelect.MenuModel()
+	dotModel: DotMenu.Model = new DotMenu.Model($r('app.media.dot_menu'))
+	dotMenuModel: DotMenu.MenuModel = new DotMenu.MenuModel()
+	dotMenu: DotMenuOption[] = [
+		new DotMenuOption('Download', 2, $r('app.media.download')),
+		new DotMenuOption('Download', 2, $r('app.media.download')),
+		new DotMenuOption('Download', 2, $r('app.media.download')),
+		new DotMenuOption(null, null),
+		new DotMenuOption(null, null, null),
+		new DotMenuOption(null, 1),
+		new DotMenuOption('Download', null),
+		new DotMenuOption('Download', null, null),
+		new DotMenuOption(null, null, $r('app.media.download')),
+		new DotMenuOption(null, 2, null),
+	]
+
+	aboutToAppear() {
+		this.selectModel.setTitle(null)
```

Comment:
should avoid setting default values or if required provide a reset method in the model and call it